resteasy-jaxrs/src/main/java/org/jboss/resteasy/plugins/providers/sse/SseEventProvider.java

```diff
@@ -47,7 +48,7 @@ public void writeTo(OutboundSseEvent event, Class<?> type, Type genericType, Ann
          MediaType mediaType, MultivaluedMap<String, Object> httpHeaders, OutputStream entityStream)
          throws IOException, WebApplicationException
    {
-      Charset charset = SseConstants.UTF8;
+      Charset charset = StandardCharsets.UTF_8;
       if (mediaType != null && mediaType.getParameters().get(MediaType.CHARSET_PARAMETER) != null)
       {
          charset = Charset.forName(mediaType.getParameters().get(MediaType.CHARSET_PARAMETER));
```

Comment:
@jimma, I'm not sure how to think about this. If the spec says that "Event streams in this format must always be encoded as UTF-8.", then shouldn't creating an event with a media type that has a charset (other than UTF-8) be considered an error?

Reply:
@ronsigal  I back to look at this again and it looks we can do this : always encode as "UTF-8" and we simply ignore the other charset.  User should always use "UTF-8" charset as event spec says .  I removed other charset support.